js/contextMenus.js

```diff
@@ -407,30 +407,32 @@ function autofillTemplateInit (suggestions, frame) {
 function tabTemplateInit (frameProps) {
   const frameKey = frameProps.get('key')
   const items = []
-  items.push(
-    CommonMenu.newTabMenuItem(frameProps.get('key')),
-    CommonMenu.separatorMenuItem,
-    {
-      label: locale.translation('reloadTab'),
-      click: (item, focusedWindow) => {
-        if (focusedWindow) {
-          focusedWindow.webContents.send(messages.SHORTCUT_FRAME_RELOAD, frameKey)
+  const location = frameProps.get('location')
+  if (!(location === 'about:blank' || location === 'about:newtab' || isIntermediateAboutPage(location))) {
+    items.push(
+      CommonMenu.newTabMenuItem(frameProps.get('key')),
+      CommonMenu.separatorMenuItem,
+      {
+        label: locale.translation('reloadTab'),
+        click: (item, focusedWindow) => {
+          if (focusedWindow) {
+            focusedWindow.webContents.send(messages.SHORTCUT_FRAME_RELOAD, frameKey)
+          }
         }
-      }
-    }, {
-      label: locale.translation('clone'),
-      click: (item, focusedWindow) => {
-        if (focusedWindow) {
-          focusedWindow.webContents.send(messages.SHORTCUT_FRAME_CLONE, frameKey, {
-            openInForeground: true
-          })
+      }, {
+        label: locale.translation('clone'),
```

Comment:
Actually clone is still a valid operation as well regardless of the current page. Because it clones the full history, you might just want a copy of the history.   Reload isn't needed for now, but it should be for example an intermediate page is also an error page for things like a network connection error.